docs/source/getting-started.md

```diff
@@ -76,23 +76,39 @@ conda uninstall jupyter_enterprise_gateway
 
 ### Installing Kernels
 
-Please follow the link below to learn more specific details about how to install/configure specific kernels with Jupyter Enterprise Gateway:
+To leverage the full distributed capabilities of Spark, Jupyter Enterprise Gateway has provided deep integration with various resource managers. Having said that, Enterprise Gateway also supports running in a pseudo-distributed mode utilizing for example both YARN client or Spark Standalone modes. We've also recently added Kubernetes, Docker Swarm and IBM Spectrum Conductor integrations.
 
-* [Installing and Configuring kernels](getting-started-kernels.html)
+Please follow the links below to learn specific details about how to enable/configure the different modes of distributing your kernels:
 
-### Configuring Resource Managers
+* [Vanilla](kernel-vanilla.html)
+* [YARN Cluster Mode](kernel-yarn-cluster-mode.html)
+* [YARN Client Mode](kernel-yarn-client-mode.html)
+* [Standalone](kernel-spark-standalone.html)
+* [Kubernetes](kernel-kubernetes.html)
+* [Docker Swarm](kernel-docker.html)
+* [IBM Spectrum Conducto](kernel-conductor.html)
 
-To leverage the full distributed capabilities of Spark, Jupyter Enterprise Gateway has provided deep integration with the Apache Hadoop YARN resource manager. Having said that, Enterprise Gateway also supports running in a pseudo-distributed mode utilizing both YARN client or Spark Standalone modes. We've also recently added Kubernetes, Docker Swarm and IBM Spectrum Conductor integrations.
+In each of the resource manager sections, we set the `KERNELS_FOLDER` to `/usr/local/share/jupyter/kernels` since that's one of the default locations searched by the Jupyter framework.  Co-locating kernelspecs hierarchies in the same parent folder is recommended, although not required.
 
-Please follow the links below to learn specific details about how to enable/configure the different modes of distributing your kernels:
+**Important requirements regarding the nodes**
+
+We have two cases:
 
-* [Enabling YARN Cluster Mode support](kernel-yarn-cluster-mode.html)
-* [Enabling YARN Client Mode or Spark Standalone support](kernel-yarn-client-mode.html)
-* [Enabling Kubernetes Support](kernel-kubernetes.html)
-* [Enabling Docker Swarm Support](kernel-docker.html)
-* [Enabling IBM Spectrum Conductor Support](kernel-conductor.html)
+1. The kernel is run in (Docker) container: in that case, the image should ensure the availability of the kernel libraries and kernelspec. There is no additional action to take on the node.
```

Comment:
in a (Docker) container.

The kernelspec is not necessary here, only the launcher.  We talk about this in container customization.